Engine/Fuego/Application.cpp

```diff
@@ -28,31 +28,24 @@ Application::Application() : d(new ApplicationImpl())
     d->m_Running = true;
 
     // Temp::
-    float vertices[3 * 3] = {
-        -0.5f, -0.5f, 0.0f, 0.5f, -0.5f, 0.0f, 0.0f, 0.5f, 0.0f};
+    float vertices[3 * 3] = {-0.5f, -0.5f, 0.0f, 0.5f, -0.5f, 0.0f, 0.0f, 0.5f, 0.0f};
     d->_vertexBuffer.reset(VertexBuffer::Create(vertices, sizeof(vertices)));
 
     uint32_t indices[3] = {0, 1, 2};
-    d->_indexBuffer.reset(
-        IndexBuffer::Create(indices, sizeof(indices) / sizeof(uint32_t)));
+    d->_indexBuffer.reset(IndexBuffer::Create(indices, sizeof(indices) / sizeof(uint32_t)));
 }
 
 Application::~Application() { delete d; }
 
 void Application::PushLayer(Layer* layer) { d->m_LayerStack.PushLayer(layer); }
 
-void Application::PushOverlay(Layer* overlay)
-{
-    d->m_LayerStack.PushOverlay(overlay);
-}
+void Application::PushOverlay(Layer* overlay) { d->m_LayerStack.PushOverlay(overlay); }
 
 void Application::OnEvent(Event& event)
 {
     EventDispatcher dispatcher(event);
-    dispatcher.Dispatch<WindowCloseEvent>(
-        std::bind(&Application::OnWindowClose, this, std::placeholders::_1));
-    dispatcher.Dispatch<WindowResizeEvent>(
-        std::bind(&Application::OnWindowResize, this, std::placeholders::_1));
+    dispatcher.Dispatch<WindowCloseEvent>(std::bind(&Application::OnWindowClose, this, std::placeholders::_1));
```

Comment:
May we change to smth like this?:
dispatcher.Dispatch<WindowCloseEvent>(std::bind(&Application::OnWindowClose, this,
                                                                    std::placeholders::_1));

Reply:
Sorry, didn't get this one